lib/sentry/config.ex

```diff
@@ -58,6 +58,14 @@ defmodule Sentry.Config do
               Whether to enable the Oban integration. When enabled, the Sentry SDK will
               capture check-ins for Oban jobs. *Available since v10.2.0*.
               """
+            ],
+            monitor_name_generator: [
+              type: :mfa,
```

Comment:
Since this is forced to accept only a single argument, make this:

```suggestion
              type: {:tuple, [:atom, :atom]},
              type_doc: "`{module(), atom()}`",
```

so that you can't accidentally call it in a wrong way.